usagov/usagov-2021
# USAGOV-1475 use uswds for font family and font weight tokens

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1475

## Description

Went through the sass files and changed the font-families and font-weights to use uswds tokens.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- Check a couple different pages (especially the federal directories index) and the footer to make sure no changes are present in computed values of font-family and font-weights. 

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
